scripts/dev-environment-setup.sh

```diff
@@ -383,43 +389,29 @@ install_aws_cli () {
     fi
 }
 
-install_databases () {
-    SKIP="$(check_homebrew)"
+start_database() {
+  # Check if mongodb is installed
+  brew ls --versions mongodb > /dev/null
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** What is the behaviour when `brew` is not installed on the developer's system? I think we should handle cases where it is not in a silent way.‏